obs-studio-server/source/nodeobs_display.cpp

```diff
@@ -89,7 +89,8 @@ struct DestroyWindowMessageQuestion {
 	HWND window;
 };
 
-struct DestroyWindowMessageAnswer : message_answer {};
+struct DestroyWindowMessageAnswer : message_answer {
+};
```

Comment:
this change does not look necessary for a PR.

Reply:
This is done by clang-format. Should I cancel the clang-format commit?

Reply:
strange what this clang format fix was not done by Stevens PR.